src/admin/python/lsst/qserv/admin/cli/options.py

```diff
@@ -237,3 +237,36 @@
 below. The option name should NOT include prefix dashes.""",
     callback=yaml_presets,
 )
+
+
+cmd_option = partial(
+    click.option,
+    "--cmd",
```

Comment:
We are migrating toward as much execution-time configurability as possible. This allows more configurability without having to rebuild the run image. We hope that this will make development and debugging easier in some cases, and enable more kinds of emergency patches.